build_tools/cmake/test.sh

```diff
@@ -27,23 +27,30 @@ export CTEST_PARALLEL_LEVEL=${CTEST_PARALLEL_LEVEL:-$(nproc)}
 export IREE_LLVMJIT_DISABLE=${IREE_LLVMJIT_DISABLE:-1}
 export IREE_VULKAN_DISABLE=${IREE_VULKAN_DISABLE:-1}
 
-EXCLUDED_TESTS=(
-    iree/compiler/Translation/SPIRV/LinalgToSPIRV/test:pw_add.mlir.test
-    iree/hal/vulkan:dynamic_symbols_test
-    iree/test/e2e/xla:rem.mlir.test
-    bindings_python_pyiree_rt_function_abi_test
-    bindings_python_pyiree_rt_system_api_test
-    bindings_python_pyiree_rt_vm_test
-    bindings_python_pyiree_rt_hal_test # TODO: Enable after the VM is fixed
-    bindings_python_pyiree_compiler_compiler_test # TODO: Enable after the VM is fixed
-    # TODO(b/146898896) get label-based exclusions working
-    iree/modules/check/test:check_success.mlir_vulkan-spirv_vulkan
-    iree/modules/check/test:check_failure_failure.mlir_vulkan-spirv_vulkan
+# Tests to exclude by label. In addition to any custom labels (which are carried
+# over from Bazel tags), every test should be labeled with the directory it is in.
+declare -a label_exclude_args=(
+  # Exclude specific labels.
+  # Put the whole label with anchors for exact matches.
+  # For example:
+  #   ^driver=vulkan$
+  ^driver=vulkan$
+  ^nokokoro$
+  # Exclude all tests in a directory.
+  # Put the whole directory with anchors for exact matches.
+  # For example:
+  #   ^bindings/python/pyiree/rt$
+  ^bindings$
+  # Exclude all tests in some subdirectories.
+  # Put the whole parent directory with only a starting anchor.
+  # Use a trailing slash to avoid prefix collisions.
+  # For example:
+  #   ^bindings/
+  ^bindings/
```

Comment:
Can we already add labels to the tests that will fail in the VM, thus that we can safely allow to run tests on `bindings`?

Reply:
Yes we could individually label those tests. Since they all seemed to be failing I just excluded the entire directory. I can label them all "nokokoro" instead though.

Reply:
As discussed on discord, please let's add a TODO somewhere that we need to renable to two python tests that can pass in the VM (after the VM itself got fixed).

Reply:
Done.